### PR TITLE
10483: Display Empty Message to Public Users;

### PR DESCRIPTION
**Pull request by cruzjone-flexion**

### CODE REVIEW

**Review by zachrog — APPROVED** [review]

File: web-client/src/presenter/state-public.ts [hunk 1]
```diff
@@ -27,7 +27,9 @@ const computeds = {
     typeof practitionerSearchHelper
   >,
   publicCaseDetailHeaderHelper,
-  publicCaseDetailHelper,
+  publicCaseDetailHelper: publicCaseDetailHelper as unknown as ReturnType<
+    typeof publicCaseDetailHelper
+  >,
```
Comment (Mwindo) [inline web-client/src/presenter/state-public.ts:32]: Is the type not inferred here properly without the `as unknown as ReturnType<...`?

**Review by Mwindo — APPROVED** [review]

File: web-client/src/presenter/state-public.ts (hunk 1 above)
Comment (zachrog) [reply web-client/src/presenter/state-public.ts:32]: This is what we have to do for all helpers for them to appear correct in the .tsx files

Comment (Mwindo) [reply web-client/src/presenter/state-public.ts:32]: Thank you!

Comment (cruzjone-flexion) [reply web-client/src/presenter/state-public.ts:32]: No, helper is a function and we want to type it as the results of the function.

**Pull request merged**